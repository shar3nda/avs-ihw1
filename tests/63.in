13
6 9 6 -2 0 7 2 7 2 -2 7 10 2