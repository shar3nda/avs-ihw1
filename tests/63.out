-2 0 7 2 7 2 -2 7 10 2